### PR TITLE
HARP-7831 Webapp optimization

### PR DESCRIPTION
**Pull request by zbigg**
harp.gl website refactor
    
Optimizing harp.gl wbsite for speed.
    
    * Moved `harp.gl` webpage to `www` folder
    * Refactored harp.gl app to be "proper" webpack app
    * Reduce number or requestd by bundling JS into one file and CSS into
    HTML.
    * <link rel="preload"> key resources

harp.gl www - cleanup theme
    
    * Remove 'text' techniques as we don't want text in this theme.
    * Migrate unsupported nested styles to flat form.

Related-to: HARP-7831

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+(Looks like) Cloudfront is the frontend for `harp.gl`, works as proxy for
+1) `https://www.harp.gl/docs/*` - rev-proxied to S3
+2) rest - rev-prexied to `https://heremaps.github.io/harp.gl`
+
+## Production Deployment
+
+(all paths relative to `harp.gl` project root)
+
+1) `harp.gl` site is static and lives in `docs/index/html`. It uses `harp.gl` bundle and three.js
```
Comment (ninok) [inline www/README.md]: That's (hopefully)  not true anymore (I mean using the harp.gl bundle from unpkg.com)

Comment (zbigg) [reply www/README.md]: Yep, ... it's just doc in progress - i was documenting how it worked. This patch fixes it, i will remove outdated WIP docs.

File: www/css/index.css [hunk 2]
```diff
@@ -52,9 +52,10 @@ header {
    width: 100%;
    height: var(--header-height);
    background: var(--background);
-   background-image: '';
-   background-size: contain;
+   background-image: url('../resources/background.jpg');
```
Comment (ninok) [inline www/css/index.css]: I saw there is also a background.png. Any reason for that?

Comment (zbigg) [reply www/css/index.css]: Removed png.

**Comment by zbigg** [comment]
Rebuild please

File: www/css/index.css [hunk 3]
```diff
@@ -282,9 +284,9 @@ footer {
    }
 
    #tutorial-nav {
-      display: none;
+      /* display: none; */
```
Comment (musculman) [inline www/css/index.css]: Could be simply removed, right?

File: www/package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@here/harp-www",
```
Comment (musculman) [inline www/package.json]: perhaps harp-website would be a better name? Maybe more easy to remember?

File: www/css/index.css (hunk 3 above)
Comment (zbigg) [reply www/css/index.css]: Removed, thanks.

File: www/package.json (hunk 4 above)
Comment (zbigg) [reply www/package.json]: Applied.

**Pull request merged**